src/review/views/review_view.py

```diff
@@ -34,6 +35,10 @@ def create(self, request, *args, **kwargs):
         unified_document = ResearchhubUnifiedDocument.objects.get(id=args[0])
         request.data["created_by"] = request.user.id
         request.data["unified_document"] = unified_document.id
+        request.data["content_type"] = ContentType.objects.get(
+            model=request.data.get("content_type", None)
```

Comment:
We should have an allowed list of accepted content types, so a user won't be able to create a review of another model. The bounty view set (line 179) should have an example.